bom-weekly/pom.xml

```diff
@@ -939,7 +939,7 @@
       <dependency>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>ssh-credentials</artifactId>
-        <version>326.v7fcb_a_ef6194b_</version>
+        <version>322.v124df57ed808</version>
```

Comment:
Did not take the necessary actions to prevent future updates from being automatically merged as in https://github.com/jenkinsci/bom/pull/3052.

Reply:
Thanks.  I'll submit a pull request proposing the necessary change.